api/helpers/validation/customerValidation.js

```diff
@@ -81,6 +81,12 @@ class CustomerValidation {
             throw new UpdateError('One or more updated fields are empty.');
         }
     }
+
+    static ValidateIdExists(id) {
```

Comment:
from the code where you use this method it seems like you are not passing ID but a result of Fetch, so this name is a bit confusing

Reply:
and actually is this needed here? I'm not sure if this kind of method should be in a Validation layer as for now we were using Validation layer to Validate request. Now you use this method in the domain to check if fetch worked as expected. Maybe it should be moved to a domain itself as a separate private method or moved out to some separate class in a Domain folder

Reply:
Actually I'm using it in Repository not Domain. I can move it if you want that but honestly it looks clean to me. So, your call.